### DISCUSSION TITLE
FluentDataGrid Column Virtualization

### DISCUSSION THREAD

**Discussion by bLinMil** [microsoft/fluentui-blazor#2953] [Ideas]
Hello,

I have a Data Grid that could potentially have a lot ~100 columns and this is quite a performance hit. I use virtualization and an items provider for the rows which works great.

Would it be possible to add a native ColumnVirtualize="true" for example to the data grid?

Thank you

**Comment by vnbaaij** [comment]
No, this is not possible. The Blazor Virtualize capability (which we use) only looks at height of an area and the items in it, not at the width.
